features/example_groups/shared_context.feature

```diff
@@ -1,15 +1,23 @@
 Feature: shared context
 
-  Use `shared_context` to define a block that will be evaluated in the context
-  of example groups either explicitly, using `include_context`, or implicitly by
-  matching metadata.
+  Use `shared_context` to define a block that will be evaluated in the context of example groups either locally, using `include_context` in an example group, or globally using `config.include_context`.
```

Comment:
Is the non broken up line for relish formatting? It's a real shame there isn't a better way...